index.html

```diff
@@ -10,8 +10,194 @@
       href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
       rel="stylesheet"
     />
-    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
-    <title>Creative Commons Global Summit 2015</title>
+
+    <link rel="stylesheet" href="style.css" />
+    <title>Europe Global Summit 2023</title>
   </head>
-  <body></body>
+  <body>
+    <section class="main">
+      <div class="mobile-menu">
+        <img
+          src="./images/ClosingIcon.png"
+          alt="closing icon"
+          class="closing-icon"
+        />
+        <a href="index.html" class="logo-imgs"
+          ><img src="images/earth-globe.png" class="mobile-logo" alt="logo"
+        /></a>
+        <ul class="navigation">
+          <li class="menu"><a href="about.html" class="hover">About</a></li>
+          <li class="menu"><a href="#program" class="hover">Program</a></li>
+          <li class="menu"><a href="#join" class="hover">Join</a></li>
+          <li class="menu"><a href="#sponsor" class="hover">Sponsor</a></li>
+          <li class="menu"><a href="#news" class="hover">News</a></li>
+        </ul>
+      </div>
+      <div class="nav-left">
+        <img src="images/enabled.jpg" alt="Enable icon" class="enabled-menu" />
+      </div>
+      <header class="desktop-menu">
+        <div class="pre-header">
+          <div class="pre-header-flex">
+            <ion-icon name="logo-facebook" class="icon"></ion-icon>
+            <ion-icon name="logo-twitter" class="icon"></ion-icon>
+            <span class="tabs">English</span>
+            <span class="tabs">Language</span>
+            <span class="tabs">Logout</span>
+          </div>
+        </div>
+        <div class="header">
```

Comment:
- [ ] You did a good job 😃, but I see that you already aligned the navs to the right, but you aligned it more than expected, so please as stated by the previous reviewer try to align the content of the navs to the right little to be like the Figma Design💯

> Check below to see the difference 😊⤵️
> Click on the image to open it at a new tab in the browser if you can not see it clearly 😊

| Figma design | Your design| 
| --- | --- |
|    ![image](https://user-images.githubusercontent.com/83360636/217302469-c35f9be5-2514-4d28-bee9-8f2ec722a69c.png) |    ![image](https://user-images.githubusercontent.com/83360636/217302348-04f6ac8f-7695-4fd4-8cdc-5f48350284e1.png)|